lib/twiglet/message.rb

```diff
@@ -0,0 +1,29 @@
+module Twiglet
+  class Message
```

Comment:
Why not have Message inherit from Hash, that way we get merge and to_s for free?
That would also mean that we wouldn't need `format`.